### ISSUE TITLE
Opening and sharing improvements

### ISSUE THREAD

**Issue by bb1a2** [mortalis13/HomeDictionary#5]
- Be available as a popup like GoldenDict, Fora, Colordict, etc.
- Be available through sharing (copy word in another app and share to Home Dictionary)

**Comment by mortalis13** [comment]
"Available as a popup"?
Like "open with"?

**Comment by bb1a2** [comment]
Yes. With some apps like many epub readers, they allow you to select a word in the book and open the definition in a dictionary app, usually (but not necessarily) in a small popup. Another way is to share a word through the regular Android share method.
